.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"editor.formatOnSave": false
```

Comment:
Now that we have Prettier configured here, we _want to_ use `formatOnSave`. `prettier-config` has a good [example VS Code configuration](https://github.com/6RiverSystems/prettier-config/blob/a117d4ed9edb060af00ddca6532bfb2d8c01cd93/.vscode/settings.json).